src/components/CDPList.js

```diff
@@ -74,25 +158,95 @@ const CDPList = memo(function({ currentPath, viewedAddress, currentQuery }) {
     }
   }, [account, navbarCdps, cdps, feeds, viewedAddress]);
 
+  const [scrollPosition, setScrollPosition] = useState({
+    scrollTop: 0,
+    maxScrollTop: 1
+  });
+  const { scrollTop, maxScrollTop } = scrollPosition;
+
+  const cdpContainerRef = useRef(null);
+
+  const handleOnScroll = () => {
```

Comment:
is this function for determining when to hide the directional buttons?

Reply:
Yeah, and to keep track of the scroll position so when clicking the directional buttons, they advance from the current position. IIRC the `useRef` wasn't updating the `current.scrollTop` property after scrolling with the mouse, but I'll double check this.